earth_enterprise/src/common/tests/SConscript

```diff
@@ -50,8 +50,16 @@ map(lambda test: env.test(test, [test + '.cc'],
     ['etencoder_tests',
      ])
 
+testdir = os.path.join(env.exportdirs['bin'],'tests')
+env.copyfile(testdir, 'RunAllTests.pl')
 
-env.copyfile(os.path.join(env.exportdirs['bin'],'tests'),
-             'RunAllTests.pl')
+# Drop in the test data
+env.Command(
```

Comment:
Chris,
maybe put copying of script into the command as well.
So, we will have the command that manages everything we need to run tests.

could be something like this:

\# put this somewhere on the top of file. I am expecting it should be a path in bin../.. 
current_dir = Dir('.').abspath    
...
testdir = os.path.join(env.exportdirs['bin'],'tests')

\# Copy the test data and the script that run all tests.
test_target = '%s/.install_for_test' % current_dir
env.Command(
    test_target, testdir,
    [env.copyfile(testdir, 'RunAllTests.pl'), 
     env.rsync_cmd % (Dir('#fusion/testdata').abspath, '%s/tests/fusion/' % Dir(env.exportdirs['bin']).abspath),
     Touch('$TARGET')])